test/libsolidity/smtCheckerTests/functions/constructor_hierarchy.sol

```diff
@@ -1,6 +1,16 @@
 pragma experimental SMTChecker;
-contract C { constructor(uint) public {} }
-contract A is C { constructor() C(2) public {} }
+contract C {
+	uint a;
+	constructor(uint x) public {
+		a = x;
+	}
+}
+
+contract A is C {
+	constructor() C(2) public {
+		assert(a == 2);
+		assert(a == 3);
+	}
+}
 // ----
-// Warning: (45-72): Assertion checker does not yet support constructors.
-// Warning: (93-121): Assertion checker does not yet support constructors.
+// Warning: (166-180): Assertion violation happens here
```

Comment:
It would be really cool if we could somehow make github highlight or jump to the source locations for these - I think that every time on your SMT checks ;-). But I trust it's the correct one.

Reply:
Actually not for this PR, but generally: do you think it'd be good to repeat the ``SourceLocation.text()`` of the entire assertion?
Like ``Assertion "assert(blablabla)" failed here``?

Reply:
we could write a small browser plugin to provide that highlighting functionality

Reply:
@ekpyron good idea